config/rhoai/binding_admin_roles.yaml

```diff
@@ -0,0 +1,16 @@
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: kueue-admin-rolebinding
```

Comment:
Theres a `kueue-` prefix applied in [rhoai/kustomization.yaml](https://github.com/opendatahub-io/kueue/blob/dev/config/rhoai/kustomization.yaml#L9). So this may be named `admin-rolebinding`. When applied the prefix with be added resulting in `kueue-admin-rolebinding` as opposed to currently, which it will be named `kueue-kueue-admin-rolebinding`